artcommon/artcommonlib/util.py

```
@@ -401,6 +401,12 @@ def get_inflight(assembly, group, date=None):
         f'{RELEASE_SCHEDULES}/{prev_group}.z/?fields=all_ga_tasks',
         headers={'Accept': 'application/json'},
     )
+    if response.status_code == 404:
+        logger.info(
```

Ruff (0.16.3)

[error] Undefined name `logger`

(F821)